rxv/rxv.py

```diff
@@ -248,6 +250,41 @@ def inputs(self):
         return self._inputs_cache
 
     @property
+    def outputs(self):
+        outputs = {}
+
+        for cmd in self._find_commands('System,Sound_Video,HDMI,Output'):
+            # An output typically looks like this:
+            #   System,Sound_Video,HDMI,Output,OUT_1
+            # Extract the index number at the end as it is needed when
+            # requesting its current state.
+            m = re.match(r'.*_(\d+)$', cmd)
+            if m is None:
+                continue
+
+            port_number = m.group(1)
+            request = HdmiOut.format(port=port_number, command='GetParam')
+            response = self._request('GET', request, zone_cmd=False)
+            port_state = response.find(cmd.replace(',', '/')).text.lower()
+            outputs['hdmi' + str(port_number)] = port_state
+
+        return outputs
+
+    def set_output_enabled(self, port, enabled):
```

Comment:
Can we change `set_output_enabled` to `enable_output`?  I really don't like `set_` prefixes. Other than that it looks OK to me, but I'm not able to check it IRL since my receiver does not support those features.